action/show.go

```diff
@@ -20,36 +19,34 @@ func (s *Action) Show(c *cli.Context) error {
 		return s.List(c)
 	}
 
+	if c.Bool("clip") {
```

Comment:
Not a first sight, but clip and dmenu use the clipboard in the background. We only want to copy the first line of the secret into the clipboard with `gopass show -c` and also with `gopass completion dmenu`. So instead of doing the first line extraction inside the `copyToClipboard` func is moved it to the store with a `func First(name string)` that returns only the first line of the secret.